src/pretix/control/views/waitinglist.py

```diff
@@ -72,6 +72,27 @@ def post(self, request, *args, **kwargs):
                     'organizer': request.event.organizer.slug
                 }))
 
+        elif 'delete' in request.POST:
+            if not request.user.has_event_permission(request.organizer, request.event, 'can_change_orders'):
+                messages.error(request, _('You do not have permission to do this'))
+                return redirect(reverse('control:event.orders.waitinglist', kwargs={
+                    'event': request.event.slug,
+                    'organizer': request.event.organizer.slug
+                }))
+            try:
+                wle = WaitingListEntry.objects.get(
+                    pk=request.POST.get('delete'), event=self.request.event,
+                )
+                wle.delete()
+                messages.success(request, _('Entry is successfully deleted.'))
```

Comment:
More consistant with the rest of the system would be "The selected entry has been deleted"